### PR TITLE
New in Chrome 90

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- New in Chrome 90 post

### CODE REVIEW

File: site/en/blog/new-in-chrome-90/index.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
+---
+title: New in Chrome 90
+description: >
+  Chrome 90 is rolling out now! There's a new value for the CSS overflow property. The Feature Policy API has been renamed to Permission Policy. And there's a new way to implement and use Shadow DOM directly in HTML. Plus there's plenty more.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/blog/new-in-chrome-90/index.md:4]: https://w3c.github.io/webappsec-permissions-policy/

File: site/en/blog/new-in-chrome-90/index.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+Using `overflow: clip` makes it possible for you to prevent any type of
+scrolling for the box, including programmatic scrolling. That means the box
+is not considered a scroll container; it does not start a new formatting
+context, which gives it better performance than `overflow: hidden`. And if
+you need it, you can apply clipping to a single axis via `overflow-x`
+and `overflow-y`.
+
+Oh, and FYI - there's also `overflow-clip-margin`, which allows you to expand
+the clip border. Useful for cases where there is ink overflow that should be
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:94]:
```suggestion
the clip border. This is useful for cases where there is ink overflow that should be
```

File: site/en/blog/new-in-chrome-90/index.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+Back in Chrome 74, we introduced the [Feature Policy API][feature-policy-intro],
+which allows you to selectively enable, disable, and modify the behavior of
+certain APIs and web features in the browser. These policies are a contract
+between you and the browser. They inform the browser about what your intent is.
+
+If your code, or any of the 3rd party libraries you use violate your
+preselected rules, the browser overrides the behavior with better UX or
+just says - talk to the hand - blocking the API altogether.
+
+Starting in Chrome 90, the Feature Policy API will be renamed to the
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:128]:
```suggestion
Starting in Chrome 90, the Feature Policy API will be renamed
```

File: site/en/blog/new-in-chrome-90/index.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+```html
+<host-element>
+  <template shadowroot="open">
+    <slot></slot>
+  </template>
+  <h2>Light content</h2>
+</host-element>
+```
+
+Loading the pure HTML markup results in this DOM tree.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:190]:
```suggestion
Loading the pure HTML markup results in this DOM tree:
```

File: site/en/blog/new-in-chrome-90/index.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+```html
+<host-element>
+  #shadow-root (open)
+  <slot>
+    ↳
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:196]: Should this be here?

File: site/en/blog/new-in-chrome-90/index.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+* [Chromium source repository change list](https://chromium.googlesource.com/chromium/src/+log/89.0.4389.75..90.0.4430.71)
+
+## Subscribe
+
+Want to stay up to date with our videos, then [subscribe](https://goo.gl/6FP1a5)
+to our [Chrome Developers YouTube channel](https://www.youtube.com/user/ChromeDevelopers/),
+and you'll get an email notification whenever we launch a new video.
+
+I'm Pete LePage, and as soon as Chrome 91 is released, I'll be right here to
+tell you what's new in Chrome!
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:238]:
```suggestion
tell you what's new in Chrome.
```

File: site/en/blog/new-in-chrome-90/index.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+
+## A special shout out
+
+I had a lot of fun shooting this 1990s themed episode of New in Chrome! Huge
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:242]:
```suggestion
I had a lot of fun shooting this 1990s themed episode of New in Chrome. Huge
```

File: site/en/blog/new-in-chrome-90/index.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+thanks to Sean Meehan for the idea and for bringing together the amazing
+folks helped open the time warp to 1990.
```
Comment (jpmedley) [inline site/en/blog/new-in-chrome-90/index.md:244]:
```suggestion
folks who helped open the time warp to 1990.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/blog/new-in-chrome-90/index.md (hunk 5 above)
Comment (petele) [reply site/en/blog/new-in-chrome-90/index.md:196]: I think so, I copied it directly from Jason's article.

File: site/en/blog/new-in-chrome-90/index.md (hunk 6 above)
Comment (petele) [reply site/en/blog/new-in-chrome-90/index.md:238]: I'd like to keep it for the emphasis... When you watch the video, you'll see me get a little funky at the end. :)

**Comment by petele** [comment]
Hey @jpmedley - addressed all your feedback. Thanks for the review/edits!

File: site/en/blog/new-in-chrome-90/index.md (hunk 6 above)
Comment (jpmedley) [reply site/en/blog/new-in-chrome-90/index.md:238]: I'll approve this. Please be aware that exclamation marks are not for emphasis. They're for exclamations. "Stop!" "Wow!" "Great job". Some things in speech just don't translate to text.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**